clang/lib/CConv/DeclRewriter.cpp

```diff
@@ -460,15 +460,10 @@ bool FunctionDeclBuilder::VisitFunctionDecl(FunctionDecl *FD) {
   if (!Defnc->hasBody())
     return true;
 
-  // If a function had a checked type in the input program, we don't want to
-  // mess with the existing type.
-  if (Defnc->getIsOriginallyChecked())
-    return true;
-
   // DidAny tracks if we have made any changes to this function declaration.
   // If no changes are made, then there is no need to rewrite anything, and the
   // declaration is not added to RewriteThese.
-  bool DidAny = Defnc->numParams() > 0;
```

Comment:
IIRC, we want to rewrite `void foo()` to be `void foo(void)`. Is this change going to affect that? I remember @sroy4899 changed some code to rewrite when the prototype had no arguments.

Reply:
Looks like even the current version is only rewriting `foo()` to `foo(void)` on some occasions. We can revisit later.